### ISSUE TITLE
Planned changes

### ISSUE THREAD

**Issue by MichaelHillcox** [AdvancedXRay/Auto-Clicker#67]
Just putting this down here so my plans are clear to anyone that might want to support the project

## Changes

- [ ] Multi-loader support (Fabric, NeoForge, Forge). This can be done without any extra libs (aka Architectury)
  - [ ] Move mappings to Minecraft to allow for this
- [ ] Support all keys using a new GUI that would list the keys on the left side and have the settings on a larger panel to the right (See mod menu / Forge's mod list)
  - [ ] Support key groupings
  - [ ] Support mass editing 

## Anything more?

If you have more ideas, please post them in the comments here and I'll add them as I see fit :D

**Comment by AdyTech99** [comment]
> This can be done without any extra libs (aka Architectury)

Why not use Architectury? It makes things a lot easier.

**Comment by AdyTech99** [comment]
We can still use Architectury gradle plugin without using Architectury API.

**Comment by MichaelHillcox** [comment]
Yeah, this is what I was saying. I have nothing against arch loom. The tool chain is the least of my concern xD

**Comment by Davsennn** [comment]
Can you add an argument that stops clicking when the held tool is at a low durability?

**Comment by Davsennn** [comment]
And I just wanted to say that this mod can be used out-of-focus by toggling F3+P (Pause on lost focus) so the game will continue running when tabbed out, maybe say that in the description
appreciate your work <3

**Comment by AdyTech99** [comment]
> Can you add an argument that stops clicking when the held tool is at a low durability?

Will do this if it's all right with @MichaelHillcox and once the current PR gets merged.

**Comment by MichaelHillcox** [comment]
Yeah, this seems fine :D

**Comment by CatRat900** [comment]
Hello! I think this mod is great so far! I would love to see a feature for auto-eating if you get to a certain hunger level. Would make afking overnight less scary. For example, I have my sword in my main hand and a stack of food in my off hand, it'd be nice to be able to be able to auto eat (or auto use without the jumping) at a certain hunger level.

Thank you!
